src/dal/Jalasoft.TeamUp.Resumes.DAL.Interfaces/IDapperWrapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Jalasoft.TeamUp.Resumes.DAL.Interfaces
```

Comment:
we do not need this interface please remove

Reply:
I delete file